src/components/_classes/nesteddata/NestedDataComponent.js

```diff
@@ -25,6 +25,9 @@ export default class NestedDataComponent extends NestedComponent {
   }
 
   componentContext() {
+    if (this.component.type === 'resourcefields') {
```

Comment:
Shouldn't the ResourceFields component just override this method and return ```this._data```?  This seems like a hack that we will regret in the future.

Reply:
Sure. ResourceFields component from formio-app, so I've created PR with componentContext method there.
https://github.com/formio/formio-app/pull/1223
This PR can be closed.